package/boot/uboot-rockchip/Makefile

```diff
@@ -29,13 +29,13 @@ endef
 define U-Boot/rk3308/Default
   BUILD_SUBTARGET:=armv8
   DEPENDS:=+PACKAGE_u-boot-$(1):trusted-firmware-a-rk3308
-  ATF:=rk3308_bl31_v2.26.elf
-  TPL:=rk3308_ddr_589MHz_uart2_m1_v2.07.bin
+  ATF:=rk3308_bl31_v2.27.elf
+  TPL:=rk3308_ddr_589MHz_uart2_m1_v2.10.bin
 endef
 
 define U-Boot/rock-pi-s-rk3308
   $(U-Boot/rk3308/Default)
-  DEPENDS:=+PACKAGE_u-boot-$(1):trusted-firmware-a-rk3308-rock-pi-s
+  DEPENDS+= +PACKAGE_u-boot-$(1):trusted-firmware-a-rk3308-rock-pi-s-tpl
   TPL:=rk3308_ddr_589MHz_uart0_m0_v2.07.bin
```

Comment:
I asked my colleague to create rk3308_ddr_589MHz_uart0_m0_v2.10.bin and push it to radxa/rkbin.

Reply:
updated.